Src/java/cql-to-elm/src/main/java/org/cqframework/cql/cql2elm/LibraryManager.java

```diff
@@ -328,11 +328,10 @@ private TranslatedLibrary generateTranslatedLibrary(Library library) {
                 for (ExpressionDef expressionDef : library.getStatements().getDef()) {
 
                     //to do implement an ElmTypeInferencingVisitor; make sure that the resultType is set for each node
-                    if(!(expressionDef.getExpression() instanceof  SingletonFrom)) {
+                    if (!(expressionDef.getExpression() instanceof SingletonFrom)) {
                         if (expressionDef.getResultTypeSpecifier() != null || expressionDef.getResultType() != null) {
```

Comment:
I still don't understand why we're looking specficially at SingletonFrom nodes here? It's any expression